platform/marvell-arm64/platform.conf

```diff
@@ -172,8 +226,59 @@ get_install_device()
     return 1
 }
 
+remove_dev_partitions() {
+    echo "Remove all existing partitions except ONIE from ${blk_dev}"
+    local dev_to_install="${blk_dev}p"
+    for p in $(seq ${sonic_part_num} 9) ; do
+        if [[ -e ${dev_to_install}${p} ]]; then
+            echo "Removing partition ${dev_to_install}${p}"
+            sgdisk -d ${p} ${blk_dev} || true
+        fi
+    done
+    partprobe ${blk_dev}
+}
+
 create_demo_partition() {
 
+    if [ "$PLATFORM" = "$PLATFORM_AC5X" ]
```

Comment:
why creating demo partition is platform specific? can you check with marvel on IS_MASS branch? can we get this part of code cleaned up?